core.js

```diff
@@ -768,6 +778,13 @@ async function _fromTokenizer(tokenizer) {
 		};
 	}
 
+	if (check([0xC5, 0xD0, 0xD3, 0xC6])) {
```

Comment:
Please add the commented change request for other EPS types, and bump the minor version. Everything else looks great!

Reply:
The 'plaintext' EPS check is included in the PostScript check [here](https://github.com/sindresorhus/file-type/pull/353/files/1f121dcfeee45d02a4eef02ad62d8cf6d5b1907d#diff-04aff9558c8839f145fd5cc256639700R123) since they share the same first two bytes. The signature to check varies because of version numbers (`"%!PS-Adobe-X.X EPSF-X.X"` where each x is a single digit), so it only checks the fixed parts of the signature.

Reply:
> Please add the commented change request for other EPS types, and bump the minor version. Everything else looks great!

Better to leave the actual version increment to @sindresorhus.

Reply:
Yes, revert the version bump. You should generally never bump a version in a PR as it's up to the maintainers to decide when to release a new version and with what version (for example, they might be planning some breaking changes).

Reply:
@sindresorhus 👍 reverted